_data/associations.yml

```
@@ -11,91 +13,103 @@
 
 # Linjeforeningene
 progsys:
+  type: active
   name:     "ProgSys"
-  real:     "Linjeforeningen for studiet Informatikk: programmering og systemarkitektur"
+  real:     "Ifi-progsys"

```

IFI-ProgSys
https://2i.fi/foreninger/ProgSys_Vedteker%202018.pdf

---

Trist at de skriver forkortelsen for instituttet feil. :cry: 

```suggestion
  real:     "IFI-ProgSys"
```